### PR TITLE
PaymentProgress

### PR DESCRIPTION
**Pull request by debragordon**
## Description
throwing up here what I accomplished in case anyone wants to review, advise, use it.

This is NOT READY TO MERGE ... just wanted you to have access to it

### CODE REVIEW

File: BangazonTerminalInterface/Controllers/PaymentController.cs [hunk 1]
```diff
@@ -8,5 +9,31 @@ namespace BangazonTerminalInterface.Controllers
 {
     class PaymentController
     {
+        string paymentType; //To Capture the payment type
+        int paymentAccountNumber; //To capture the account number
+        string newPayment; //To build the new payment that will be added to the DB
+        
+        private void requestPaymentType() // Ask for payment Type
```
Comment (justinal64) [inline BangazonTerminalInterface/Controllers/PaymentController.cs:16]: @debragordon A small suggestion but I would suggest trying to use WriteToConsole() inside of the CustomerController.cs for the Console.WriteLine. This is the function I wrote(with Dustins help) to Print out generic info to the console and return the users response. Other than that everything looks good in your PaymentController.

File: BangazonTerminalInterface/Controllers/PaymentController.cs [hunk 2]
```diff
@@ -8,5 +9,31 @@ namespace BangazonTerminalInterface.Controllers
...
+            PaymentTypeValid repo = new PaymentTypeValid();
+            Console.WriteLine("Enter Payment Type" + "\n"
+                    + "> ");
+            paymentType = Console.ReadLine();
+        }
+        
+        ValidatePaymentType(string paymentType); // Send Reponse to Validator
+
+       
+        private void requestPaymentAccountNumber() // Ask for account number
```
Comment (justinal64) [inline BangazonTerminalInterface/Controllers/PaymentController.cs:27]: @debragordon same as above.

File: BangazonTerminalInterface/Controllers/PaymentController.cs (hunk 1 above)
Comment (debragordon) [reply BangazonTerminalInterface/Controllers/PaymentController.cs:16]: yes, I heard you guys talking last night about a helper function, that makes a lot of sense ... will refactor!

Comment (justinal64) [reply BangazonTerminalInterface/Controllers/PaymentController.cs:16]: We probably want to talk about a better place to store that method on Saturday.

**Pull request closed without merging**